models/event.js

```diff
@@ -0,0 +1,55 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const eventSchema = new Schema({
+  eventName: { 
+    type: String,
+    trim: true,
+    unique: true, // We would want a '_id' right? Is this what generates it? This would cause event names to be unique which we don't necessarily want...?
+    required: [true, 'Event must have a name']
+  },
+  description: { 
+    type: String,
+    trim: true, 
+  },
+  organizer: { // How should we want the organizer? Will this be populated from the name user object when the user creates the event? (concat of first/last name? or uniqueID?)
+    type: String,
+    trim: true
+  },
+  location: {
+    address: {
+      type: String,
+      trim: true
+    },
+    city: {
+      type: String,
+      trim: true
+    },
+    state: {
+      type: String,
+      trim: true
+    },
+    latitude: { 
+        type: String, // Looks like mongoose supports 'Number' as integers but not float here. Because of this and having a negative float we can just parseFloat when consuming this. 
+    },
+    longitude: {
+        type: String,
+    }
+  },
+  date: {
+    type: Date,
+    default: Date.now
+  },
+  startTime: {
+      type: String, // As string?
```

Comment:
I think that should be fine since we just need to display and won't do calculations on it. If we ever do we could convert it to use this https://stackoverflow.com/questions/15780415/how-can-i-store-time-of-day-in-mongodb-as-a-string-give-arbitrary-year-month-d